.gitmodules

```diff
@@ -36,7 +36,7 @@
 	branch = px4_firmware_nuttx-10.1.0+
 [submodule "Tools/flightgear_bridge"]
 	path = Tools/flightgear_bridge
-	url = https://github.com/PX4/PX4-FlightGear-Bridge.git
+	url = https://github.com/ThunderFly-aerospace/PX4-FlightGear-Bridge.git
```

Comment:
I will also return this to its original form before the finalizing PR. This modification is due to development, because we have a modification in the FG bridge that resends the RPM from FlightGear to PX4. (This should be merged in the bridge repository in the own PR)